src/types/ErrorsEnum.ts

```diff
@@ -0,0 +1,6 @@
+export enum ErrorsEnum {
```

Comment:
you should not add the type of the value to its name. It is obvious that we do have enum here, so the part `enum` does not help me. I would recommend to use `ErrorType`, but probably you can come up with better one